README.md

```diff
@@ -13,7 +13,19 @@ npm run build
 
 ## Android
 ### Updating the PACECloudSDK
+1. Open the `/android` folder via Android Studio
+2. In Android Studio the `build.gradle` (project level) file
+3. Adjust the line `implementation "cloud.pace:sdk:$version"` with the desired version
+4. Click on `Sync now` to fetch the new version
+
 ### Making Code Changes
+1. Head over to `/android/src/main/java/cloud/pace/plugins/cloudsdk`
+2. This folder contains all neccessary `.kt` files
+3. To start the project in Android Studio, open the `/android` folder via Android Studio, all necessary files are in `/java/cloud.pace.plugins.cloudsdk`
```

Comment:
This sections reads a bit strange due the enumeration. I think this can be written as running text.